README.md

```diff
@@ -67,6 +67,8 @@ There's a multitude of settings you can use to control the plugin server. Use th
 | INTERNAL_MMDB_SERVER_PORT     | port of the internal server used for IP location (0 means random) | `0`                                   |
 | DISTINCT_ID_LRU_SIZE          | size of persons distinct ID LRU cache                             | `10000`                               |
 | PLUGIN_SERVER_IDLE            | whether to disengage the plugin server, e.g. for development      | `false`                               |
+| POSTHOG_PROJECT_API_KEY       | PostHog project API key for analytics                             | `sTMFPsFhdP1Ssg`                      |
+| POSTHOG_API_HOST              | address of the PostHog instance used for analytics                | `https://app.posthog.com`             |
```

Comment:
It's not totally clear from the descriptions what these are used for... and if or why we would want them to be anything other than the hardcoded production values, except perhaps locally when testing. Having them exposed like this makes it very easy to opt-out of telemetry... which maybe we want, but then it should be handled in a similar way as in the main posthog app